scripts/valhalla_build_config

```diff
@@ -355,7 +355,7 @@ help_text = {
     'service': {
       'proxy': 'IPC linux domain socket file location'
     },
-    'max_reserved_labels_count': 'Maximum capacity for edge labels reserved in path algorithm'
+    'max_reserved_labels_count': 'Maximum capacity for edge labels reserved in path algorithm',
```

Comment:
this was fixed in https://github.com/valhalla/valhalla/commit/4d9d106608b354d60f4d77b4e3501d6692bbe514

Reply:
@mandeepsandhu thanks